cla_public/apps/checker/tests/test_sessions.py

```diff
@@ -0,0 +1,226 @@
+import unittest
+import uuid
+from flask import json
+from base64 import b64encode
+from flask._compat import text_type
+from werkzeug.http import http_date
+from datetime import datetime
+from cla_public.apps.checker.session import CheckerTaggedJSONSerializer, CheckerSessionObject, MeansTest
+
+
+class TestCheckerSession(unittest.TestCase):
+    def setUp(self):
+        self.serializer = CheckerTaggedJSONSerializer()
+
+    def assert_json(self, value):
+        try:
+            json.loads(value)
+        except Exception:
+            self.fail("Not valid JSON")
+
+    def format_json(self, value):
+        return json.dumps(value, separators=(",", ":"))
+
+    def compare_dicts(self, outputDict, expectedDict):
+        for key, value in expectedDict.items():
+            if isinstance(value, dict):
+                self.compare_dicts(outputDict[key], value)
+            else:
+                self.assertEqual(outputDict[key], value)
+
+    def test_serializer_with_checker_session_object(self):
+        value = CheckerSessionObject({"checker": {"foo": "bar"}})
+        outputJSON = self.serializer.dumps(value)
+        expectedDict = {" ch": {"checker": {"foo": {" b": b64encode("bar").decode("ascii")}}}}
+        expectedJSON = self.format_json(expectedDict)
+        self.assert_json(outputJSON)
+        self.assertEqual(outputJSON, expectedJSON)
+
+    def test_serializer_with_means_test(self):
+        value = MeansTest()
+        outputJSON = self.serializer.dumps(value)
+        expectedDict = {
+            u" mt": {
```

Comment:
Is this leading space in ` mt` intentional

Reply:
Yes, this is how everything data type is serialized. Not sure why..